hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/repository/EntityRepository.java

```diff
@@ -0,0 +1,34 @@
+package com.hedera.mirror.web3.repository;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import java.util.Optional;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+
+import com.hedera.mirror.common.domain.entity.Entity;
+
+public interface EntityRepository extends CrudRepository<Entity, Long> {
+
+    //FUTURE WORK Coffeine cache to be added
+    @Query(value = "select ethereumNonce from entity where evmAddress = ?1 and deleted <> true", nativeQuery = true)
```

Comment:
As this is a native query and not JPQL, you need to use native column names.
```suggestion
    @Query(value = "select ethereum_nonce from entity where evm_address = ?1 and deleted <> true", nativeQuery = true)
```